spec/spec_helper.rb

```diff
@@ -193,3 +193,9 @@ def suspend_resque_workers
   # Resume the original Resque.enqueue_to behavior.
   allow(Resque).to receive(:enqueue_to).and_call_original
 end
+
+def create_archivist
+  user = create(:user)
+  user.roles << Role.create(name: "archivist")
+  user
+end
```

Comment:
This doesn't appear to be used in this PR.

Reply:
Yeah, that was left over from another PR, looks like, thanks for catching